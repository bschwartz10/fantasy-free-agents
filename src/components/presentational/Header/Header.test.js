import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'
import logoImage from './logo.png'

describe('Header component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it('should exist', () => {
    expect(wrapper).toBeDefined()
  })

  it('should show NFL Player list', () => {
    const messageDiv = wrapper.find('.header')

    expect(messageDiv).toBeDefined()
    expect(messageDiv.text()).toEqual('NFL Player List')
  })

  it('should show rotogrinders logo', () => {
    const image = wrapper.find('.header-logo')

    expect(image).toBeDefined()
    expect(image.prop('src')).toEqual(logoImage)
  })

  it('should match snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
