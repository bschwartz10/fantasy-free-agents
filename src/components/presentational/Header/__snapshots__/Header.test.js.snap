// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot with all data passed in correctly 1`] = `
<div
  className="header"
>
  NFL Player List
  <img
    alt="Logo"
    className="header-logo"
    src="logo.png"
  />
</div>
`;
